swarm/storage/netstore.go

```diff
@@ -69,7 +69,9 @@ func NewDefaultStoreParams() (self *StoreParams) {
 //this can only finally be set after all config options (file, cmd line, env vars)
 //have been evaluated
 func (self *StoreParams) Init(path string) {
-	self.ChunkDbPath = filepath.Join(path, "chunks")
+	if self.ChunkDbPath == "" {
+		self.ChunkDbPath = filepath.Join(path, "chunks")
+	}
```

Comment:
Is this the best way to handle this or does it make more send add this to `NewDefaultStoreParams ` ? https://github.com/async-la/go-ethereum/blob/872aeef800ee6368704bec7fc8108d4a235b6e19/swarm/storage/netstore.go#L61

Reply:
I think it is fine here.

Reply:
It needs to be here because the default for a path can be overridden by command line or env var later.